content-test/common/reducers/Hints.test.js

```diff
@@ -0,0 +1,17 @@
+const Hints = require("common/reducers/Hints");
```

Comment:
It might improve calling code readability if the object were named HintsReducer.